instrumentation/grape/lib/opentelemetry/instrumentation/grape/event_handler.rb

```diff
@@ -93,8 +93,8 @@ def path(endpoint)
             return '' unless endpoint.routes
 
             namespace = endpoint.routes.first.namespace
-            version = endpoint.routes.first.options[:version] || ''
-            prefix = endpoint.routes.first.options[:prefix]&.to_s || ''
+            version = endpoint.routes.first.options[:version]&.to_s
+            prefix = endpoint.routes.first.options[:prefix]&.to_s
```

Comment:
We can stop at safely navigating the string coercion without supplying a default because `parts.reject{}` below filters `nil`s.